db/queries.py

```diff
@@ -1,46 +1,87 @@
+from typing import List
+
 from db.models import Message, User, Chat
 from django.db.models import Q, Count, F
 
 
-def get_messages_that_contain_word(word: str) -> list[Message]:
-    pass
+def get_messages_that_contain_word(word: str) -> List[Message]:
+    return list(
+        Message.objects.filter(text__icontains=word)
+    )
 
 
-def get_untitled_chats() -> list[Chat]:
-    pass
+def get_untitled_chats() -> List[Chat]:
+    return list(
+        Chat.objects.filter(title__startswith="Untitled")
+    )
 
 
-def get_users_who_sent_messages_in_2015() -> list[str]:
-    pass
+def get_users_who_sent_messages_in_2015() -> List[str]:
+    return list(
+        User.objects.filter(message__sent__year=2015)
+        .values_list("first_name", "last_name")
+    )
 
 
-def get_actual_chats() -> list[Chat]:
-    pass
+def get_actual_chats() -> List[Chat]:
+    return list(
+        Chat.objects.filter(message__sent__year__gt=2020)
+    )
 
 
-def get_messages_contain_authors_first_name() -> list[Message]:
-    pass
+def get_messages_contain_authors_first_name() -> List[Message]:
+    return list(
+        Message.objects.filter(
+            text__contains=F("user__first_name")
+        )
+    )
 
 
-def get_users_who_sent_messages_starts_with_m_or_a() -> list[User]:
-    pass
+def get_users_who_sent_messages_starts_with_m_or_a() -> List[User]:
+    return list(
+        User.objects.filter(
+            Q(message__text__istartswith="a")
+            | Q(message__text__istartswith="m")
+        )
+    )
 
 
-def get_delivered_or_admin_messages() -> list[Message]:
-    pass
+def get_delivered_or_admin_messages() -> List[Message]:
+    return list(
+        Message.objects.filter(
+            Q(is_delivered="1")
+            | Q(user__username__contains="admin")
+        )
+    )
 
 
 def get_count_messages_sent_by_first_name(first_name: str) -> int:
-    pass
-
-
-def get_top_users_by_number_of_the_messages() -> list[User]:
-    pass
-
-
-def get_last_5_messages_dicts() -> list[dict]:
-    pass
-
-
-def get_chat_dicts() -> list[dict]:
-    pass
+    return Message.objects.filter(user__first_name=first_name).count()
+
+
+def get_top_users_by_number_of_the_messages() -> List[User]:
+    return list(
+        User.objects.annotate(
+            num_messages=Count("message__user_id")
+        ).order_by("-num_messages")[:3]
+    )
+
+
+def get_last_5_messages_dicts() -> List[dict]:
+    return list(
+        {
+            "from": message.user.username, "text": message.text
+        }
```

Comment:
```suggestion
        {
            "from": message.user.username, 
            "text": message.text
        }
```
or
```suggestion
        {"from": message.user.username, "text": message.text}
```

These variants have better readability